packages/perseus/src/widgets/phet-simulation/phet-simulation.test.ts

```diff
@@ -138,4 +146,36 @@ describe("phet-simulation widget", () => {
         // Assert
         expect(url).toBe(null);
     });
+
+    it("should use the mobile app fullscreen logic when rendered in a mobile app", async () => {
+        // Arrange
+        const apiOptions: APIOptions = {
+            isMobileApp: true,
+        };
+
+        // Act
+        renderQuestion(question1, apiOptions);
+        await waitFor(() => {
+            expect(
+                screen.getByTitle("Projectile Data Lab"),
+            ).toBeInTheDocument();
+        });
+
+        await waitFor(async () => {
+            const fullscreenButton = screen.getByRole("button", {
+                name: "Fullscreen",
+            });
+            // Find and click the fullscreen button once available
+            await userEvent.click(fullscreenButton);
+        });
+
+        // Assert
+        await waitFor(() => {
+            const closeButton = screen.getByRole("button", {
+                name: "Exit fullscreen",
+            });
+            // Check that the mobile app fullscreen close button is present
+            expect(closeButton).toBeInTheDocument();
```

Comment:
I COULD verify that the styles are set, but I wasn't sure if that would be necessary as they'll be applied using the same check that the `closeButton` is using.

Reply:
I think you could check if the overlay div was present (but I agree that checking styling might be overkill).  If the button is _inside_ the overlay div then you're fine with this.